AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -520,6 +533,13 @@ - (void)selectItemAtIndexPath:(NSIndexPath *)indexPath animated:(BOOL)animated s
   [super selectItemAtIndexPath:indexPath animated:animated scrollPosition:scrollPosition];
 }
 
+- (void)waitForUpdatesDuringNextLayoutPass
+{
+  ASDisplayNodeAssertMainThread();
+  [self setNeedsLayout];
```

Comment:
You're absolutely right – that was a holdover from some previous implementation I believe. Removed.